common/lib/xmodule/xmodule/templates/problem/optionresponse_hint.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
 
         Use the following example problem as a model.
 
-        >> A/an ________ is a vegetable.<<
+        >> A/an ________ is a vegetable.||You can select only one option.<<
```

Comment:
I think there should be better descriptions here. I think @lamagnifica can suggest some.